app.js

```diff
@@ -0,0 +1,66 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        // title: "",
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+})
+
+app.Views.TodoApp = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        this.newTodoField = this.$(".todo__new-todo-field");
+        this.markAllCompleteField = this.$(".mark-all-complete");
+        this.totalCount = this.$(".total-count");
+        this.remainingCount = this.$(".remaining-count");
+        this.completeCount = this.$(".complete-count");
+    }
+});
+app.Views.TodoItem = Backbone.View.extend({
```

Comment:
`TodoItemView`

Reply:
Avoided the redundant repetition of View since it was already namespaced (app.Views), but I will change